lib/compilers/fpc.js

```diff
@@ -30,6 +30,7 @@ function compileFPC(info, env) {
     var compiler = new Compile(info, env);
     compiler.supportsOptOutput = false;
     compiler.originalPostProcess = compiler.postProcess;
+    compiler.symbolcache = [];
```

Comment:
Is this used anywhere else? If it's not, I think it migh be better as a local variable to the compileFPC (IF! It's not reinitialised every time it's called, in which case I guess the only solution is to store it on the compiler..)

Reply:
Will see what I can do, wasn't too sure about the JS scopes in this case, but will experiment